examples/state_limits.py

```diff
@@ -30,23 +30,23 @@ def future_load(t, x=None):
     event = 'impact'
     (times, inputs, states, outputs, event_states) = m.simulate_to_threshold(future_load, threshold_keys=[event], dt=0.005, save_freq=1)
     
-    # Print states
-    print('Example 1')
-    for i, state in enumerate(states):
-        print('State ', i, ': ', state)
-    print()
+    # # Print states
```

Comment:
Why is this commented out

Reply:
Sorry, this was temporarily commented out while I was testing different example files against the bar - readded!